src/bindings.js

```diff
@@ -278,18 +278,23 @@ export class ComponentBinding extends Binding {
     let bindingRegExp = view.bindingRegExp()
 
     if (this.el.attributes) {
-      this.el.attributes.forEach(attribute => {
+      let attribute = null
```

Comment:
It's an old java reflex. I never declare variables in loops. I know in javascript it isn't as important but sometimes i can't help it ;)